### PR TITLE
Support `open-location` for Navie webview

### PR DESCRIPTION
**Pull request by jansorg**
Closes https://github.com/getappmap/appmap-intellij-plugin/issues/677

This PR adds support for Navie's `open-location`.
Numbers of `name:line-line` are taken as 1-based and are adjusted to the IDE's 0-based numbers.
Relative paths are looked up in the available project files to find the best match.

I tested opening regular source files and also opening an AppMap using pre-release https://github.com/getappmap/appmap-js/releases/tag/%40appland%2Fappmap-v3.137.0

This PR is supporting both variants of the `open-location` message: `open-location "path/to/file"` and `open-location "path/to/file", "/root/dir"`. I wasn't able to test the 2nd variant, but AFAIK it would need a new release with an updated Navie webview, anyway.

### CODE REVIEW

File: appland-navie/webview.js [hunk 1]
```diff
@@ -62,6 +62,7 @@ export function mountWebview() {
     });
 
     app.$on('open-install-instructions', () => vscode.postMessage({command: 'open-install-instructions'}))
+    app.$on('open-location', (path) => vscode.postMessage({command: 'open-location', path}))
```
Comment (kgilpin) [inline appland-navie/webview.js:65]: Note than in an update coming soon, the enclosing directory of the path will also be provided. So only that directory will need to be searched for the file. You could include the directory parameter in this PR and treat it as optional?

Comment (kgilpin) [reply appland-navie/webview.js:65]: Just be aware that you can’t simply join the directory to the path to identity the file. The file path location within the directory may not be known exactly (eg Jvm recordings).

Comment (kgilpin) [reply appland-navie/webview.js:65]: You can see the WIP change here - https://github.com/getappmap/appmap-js/pull/1752/files

**Review by ahtrotta — APPROVED** [review]
Looks good and worked well when I tested it manually, thanks!

**Pull request merged**